src/main/resources/application.properties

```diff
@@ -4,3 +4,6 @@ spring.datasource.url = jdbc:postgresql://tai.db.elephantsql.com/utrfwepv
 
 spring.datasource.username = utrfwepv
 spring.datasource.password = u39l9Dk_eaDUnJZwt70PsHdBJ1UmjQKD
+
+jwt.secret=javainuse
+jwt.expirationTime = 864000000
```

Comment:
There is a better way to save sensitive data.
Think about environment variables